src/components/progress-tracker/progress-tracker.pw.tsx

```diff
@@ -207,19 +208,39 @@ test.describe("Check props for ProgressTracker component", () => {
     }
   );
 
-  (["top", "bottom"] as const).forEach((labelsPosition) => {
-    test(`render component with labelsPosition is set to ${labelsPosition}`, async ({
-      mount,
-      page,
-    }) => {
-      await mount(<ProgressTrackerComponent labelsPosition={labelsPosition} />);
+  const getYValue = (locator: Locator) =>
+    locator.evaluate((element) => element.getBoundingClientRect().y);
 
-      const progressTrackerMinValElementIndex = progressTrackerMinVal(page);
-      await expect(progressTrackerMinValElementIndex).toHaveText("50%");
+  test(`should position current value label below tracker line when labelsPosition prop is "top"`, async ({
```

Comment:
LGTM 👍🏼 Just one small typo :) 

```suggestion
  test(`should position current value label above tracker line when labelsPosition prop is "top"`, async ({
```